src/biogeophys/CanopyFluxesMod.F90

```diff
@@ -1329,11 +1393,39 @@ subroutine CanopyFluxes(bounds,  num_exposedvegp, filter_exposedvegp,
             zeta(p) = zldis(p)*vkc*grav*thvstar/(ustar(p)**2*thv(c))
 
             if (zeta(p) >= 0._r8) then     !stable
-               zeta(p) = min(zetamax,max(zeta(p),0.01_r8))
+               ! remove stability cap when biomass heat storage is active 
+               if(use_biomass_heat_storage) then 
+                  ! TODO(KWO, 2022-03-15) Only for Meier2022 for now to maintain bfb with ZengWang2007
+                  if (z0param_method == 'Meier2022') then
+                     zeta(p) = min(zetamax,max(zeta(p),0.01_r8))
+                  else
+                     zeta(p) = min(100._r8,max(zeta(p),0.01_r8))
```

Comment:
Revert new lines 1396-1406 to old line 1332. That should bring back BFB in clm51 cases.

Reply:
This test 
ERI_C2_Ld9.f10_f10_mg37.I2000Clm51BgcCrop.cheyenne_gnu.clm-default
confirms that we get BFB against dev129 again. Next I have submitted the cheyenne test-suite.

Reply:
Still getting some (though fewer) diffs from the baseline. Results filtered by this command:
`./cs.status.fails | grep -v NLCOMP | grep -v 'otherwise bit-for-bit' | grep -v 'does not exist' | grep -v 'EXPECTED FAILURE' | grep FAIL`
```
SMS_D.1x1_brazil.I2000Clm51FatesSpCruRsGs.cheyenne_gnu.clm-FatesColdDryDepSatPhen BASELINE ctsm5.1.dev129: DIFF
SMS_D.1x1_brazil.I2000Clm51FatesSpCruRsGs.cheyenne_gnu.clm-FatesColdMeganSatPhen BASELINE ctsm5.1.dev129: DIFF
ERP_P36x2_Ld30.f45_f45_mg37.I2000Clm51FatesSpCruRsGs.cheyenne_intel.clm-FatesColdSatPhen BASELINE ctsm5.1.dev129: DIFF
ERS_D_Mmpi-serial_Ld5.5x5_amazon.I2000Clm51FatesRs.cheyenne_intel.clm-FatesCold BASELINE ctsm5.1.dev129: DIFF
ERS_Ly5_P144x1.f10_f10_mg37.IHistClm51BgcCrop.cheyenne_intel.clm-cropMonthOutput BASELINE ctsm5.1.dev129: DIFF
SMS.f45_f45_mg37.I2000Clm51FatesSpRsGs.cheyenne_nvhpc.clm-FatesColdSatPhen BASELINE ctsm5.1.dev129: DIFF
```
I will investigate.

Reply:
The fates tests' lnd_in files differ from the baseline (I don't know why, yet):
```
101,102d100
<  use_z0m_snowmelt = .false.
<  z0param_method = 'ZengWang2007'
168c166
<  zetamaxstable = 2.0d00
---
>  zetamaxstable = 0.5d00
```
The one non-fates test does not have the zetamaxstable difference, so I don't know why it differs from the baseline. I will investigate.

Reply:
The non-fates diffs count as roundoff I think, maybe because we're looking at output after 5 years?
The largest normalized diffs are:
```
RMS ERRH2OSNO                        2.8529E-18            NORMALIZED  5.6909E-03
RMS ERRSEB                           2.7629E-18            NORMALIZED  4.9140E-04
RMS ERRSOI                           1.1648E-14            NORMALIZED  3.7069E-05
RMS ERRSOL                           7.7001E-19            NORMALIZED  1.4330E-03
RMS ERRH2O                           8.1752E-26            NORMALIZED  1.6803E-10
```
and the rest are 1e-12 or smaller. I will bring this up at Monday's stand-up to decide whether to dig further.

Reply:
That's certainly roundoff, especially with a 5-year case. So resolving.